src/ToDoItem/ToDoItem.tsx

```diff
@@ -24,16 +27,16 @@ const ToDoItem: React.FC<ToDoItemProps> = ({
 }) => {
   return (
     <Draggable draggableId={id.toString()} index={index}>
-      {(provided: any, snapshot: any) => (
+      {(provided: DraggableProvided, snapshot: DraggableStateSnapshot) => (
```

Comment:
If you do not specify the types here, they still will implicitly be defined.
That's not a mistake, but I like an approach that requires minimum effort.